pages/osx/afplay.md

```diff
@@ -0,0 +1,19 @@
+# afplay
+
+> Command-line audio player.
```

Comment:
Is there any links we can add here, any official documents?
You can add a new line here.

`> More information: <link here>.`

Reply:
Apparently this is a built-in command, so I can't find any links other than the Man page.

Reply:
Apparently this is a built-in command, so I can't find any links other than the Man page.

Reply:
Okay.